puppet/modules/freight/templates/deploy_debs.erb

```diff
@@ -43,6 +47,17 @@ files.each do |file|
 end
 system "/usr/bin/freight-cache -v -c /home/freight/freight.conf apt/#{os}"
 
+foreman_release_dir = "#{ENV['TARGETCACHE']}/pool/#{os}/#{version}/f/foreman-release/"
```

Comment:
I'm tempted to store the path relative to the target cache in a separate var and reuse it in `fastly-purge`.

Reply:
I was thinking about that, yeah!